targets/arm64.json

```diff
@@ -0,0 +1,24 @@
+{
+	"build-tags": ["arm64"],
+	"goarch":"arm64",
+	"goos":"linux",
+	"llvm-target":"aarch64-elf",
+	"tags":"arm64",
+	"compiler":"clang",
+	"linker":"ld.lld",
+	"rtlib": "compiler-rt",
+	"cflags": [
+		"--target=aarch64-elf",
+		"-fshort-enums",
+		"-nostdlibinc",
```

Comment:
Instead of `-nostdlibinc`, you can add `"libc": "picolibc"` (on the dev branch).

Reply:
Fixed.  Required a change to the picolib library builder to get it to work.   The fix is not included in this PR.

```
$ git diff builder/picolibc.go
diff --git a/builder/picolibc.go b/builder/picolibc.go
index 31e2f05..ede39c3 100644
--- a/builder/picolibc.go
+++ b/builder/picolibc.go
@@ -12,7 +12,7 @@ var Picolibc = Library{
        name: "picolibc",
        cflags: func() []string {
                picolibcDir := filepath.Join(goenv.Get("TINYGOROOT"), "lib/picolibc/newlib/libc")
-               return []string{"-Werror", "-Wall", "-std=gnu11", "-D_COMPILING_NEWLIB", "-fshort-enums", "--sysroot=" + picolibcDir, "-I" + picolibcDir + "/tinystdio", "-I" + goenv.Get("TINYGOROOT") + "/lib/picolibc-include"}
+               return []string{"-Werror", "-Wall", "-std=gnu11", "-D_COMPILING_NEWLIB","-I"+picolibcDir+"/include", "-fshort-enums", "--sysroot=" + picolibcDir, "-I" + picolibcDir + "/tinystdio", "-I" + goenv.Get("TINYGOROOT") + "/lib/picolibc-include"}
        },
        sourceDir: "lib/picolibc/newlib/libc",
        sources: func(target string) []string {
```